repos/system_upgrade/el7toel8/actors/checkkdegnome/actor.py

```diff
@@ -0,0 +1,137 @@
+import os
+import subprocess
+import time
+
+from leapp.actors import Actor
+from leapp.models import CanUpgrade
+from leapp.tags import RhelUpgradeTag
+
+_ret_OK = 0  # return value == 0 means everything is OK
+
+
+class CheckKdeGnome(Actor):
+    """
+    Actor will check whether KDE is installed together with GNOME desktop to inform whether we can
+    inhibit the upgrade process. When both are installed, we need to inform the user that KDE will be
+    removed and GNOME will be used instead. If only KDE is installed, we want to inhibit
+    the upgrade process otherwise the user will end up without a desktop.
+    Also if both are installed, but KDE is not used as default desktop, we further check most common
+    KDE apps to inform the user that those will be removed in case the user is using them.
+    """
+    name = 'check_kde_gnome'
+    consumes = ()
+    produces = (CanUpgrade,)
+    tags = (RhelUpgradeTag,)
```

Comment:
The tags are as of now used to tell Leapp framework in which workflow and phase of the workflow the actor should be executed. 
Since this actor is for upgrades, use `IPUWorkflowTag`. IPU stands for in-place upgrade.
Since this is a check actor, use `ChecksPhaseTag` here and remove the new RhelUpgradeTag with its file please.
For more details, see https://leapp.readthedocs.io/en/latest/actor-rhel7-to-rhel8.html#including-an-actor-in-the-rhel-7-to-8-upgrade-process.